hardhat.config.ts

```diff
@@ -57,6 +57,8 @@ const externalDeployments = {
   ],
   arbitrumsepolia: ["node_modules/@venusprotocol/governance-contracts/deployments/arbitrumsepolia"],
   arbitrumone: ["node_modules/@venusprotocol/governance-contracts/deployments/arbitrumone"],
+  opsepolia: [],
+  opmainnet: [],
```

Comment:
Maybe this is the reason because the ACM contract is set to `0x0...1` in the [deployed PSR contract](https://sepolia-optimism.etherscan.io/address/0x4d3785aD3767b93231C19543AA7c9BcD098F1f15)

I suppose we would need first the npm package from the governance repo, and then redeploy this contract (after adding here the deployment folders)

Reply:
Yes, we should release the npm version for the governance repo, but for now I have redeployed the PSR with hardcoding the ACM addresses in script. [7e7a00f](https://github.com/VenusProtocol/protocol-reserve/pull/102/commits/7e7a00ff271bb562e1d361650ffe370d539a1be6)

Reply:
Updated the dependency: 6a3bddb4a350e86797e4d5a7631dbb80f3b9a0b2